digits/scheduler.py

```diff
@@ -110,37 +116,49 @@ def load_past_jobs(self):
         """
         failed = 0
         loaded_jobs = []
-        for dir_name in sorted(os.listdir(config_value('jobs_dir'))):
-            if os.path.isdir(os.path.join(config_value('jobs_dir'), dir_name)):
-                exists = False
-
+        for workspace in sorted(os.listdir(config_value('jobs_dir'))):
+            if not self.workspace_jobs.has_key(workspace):
+                self.workspace_jobs[workspace] = []
+            
```

Comment:
There's some extra whitespace here and elsewhere.
![workspace-whitespace](https://cloud.githubusercontent.com/assets/687269/11041823/9ae12b22-86c8-11e5-9c83-b9fc737c2a48.jpg)